Blimp/GCS_Mavlink.cpp

```diff
@@ -204,18 +141,6 @@ void GCS_MAVLINK_Blimp::send_pid_tuning()
         }
         const AP_Logger::PID_Info *pid_info = nullptr;
         switch (axes[i]) { //TODO This should probably become an acceleration controller?
-        // case PID_TUNING_ROLL:
```

Comment:
As mentioned in my comment on #17412, I am currently implementing this code. If you want to remove this part from the PR then the rest looks ok.
Thanks.